tests/test_parastell.py

```diff
@@ -29,28 +29,171 @@ def remove_files():
             Path.unlink(file)
 
 
+def check_cubit():
+    try:
+        import cubit
+
+        return True
+    except ImportError:
+        return False
```

Comment:
I wonder if it makes sense to put this in `cubit_utils`?  Not sure, to be honest, but seems like it could be useful...

Reply:
I think organizationally that makes sense